### ISSUE TITLE
Add relevant screen shots of the app

### ISSUE THREAD

**Issue by Bhavnaharitsa** [dsciitpatna/Shabd#163]
**Actual Behaviour**
We do not have any screenshots of the app. It is necessary to have screenshots because that's when users will get a brief idea and on how the app looks.

**Expected Behaviour**
Add screenshots and display them in README.md file with proper dimensions.

**Would you like to work on the issue?**
No. Other contributors can take it.

**Comment by goyal9314** [comment]
After sign in page app is not opening then how to take screenshots of different pages.
![image](https://user-images.githubusercontent.com/43785484/57674364-bea35000-763c-11e9-8a48-d568eca47eb2.png)

**Comment by atm1504** [comment]
@try to sign in again.

**Comment by atm1504** [comment]
@goyal9314 try to sign in again. Or try to use a different gmail account

**Comment by goyal9314** [comment]
@atm1504 can we go inside app without sign in? because i tried with 3 gmail accounts but it is showing login failed.

**Comment by atm1504** [comment]
> @atm1504 can we go inside app without sign in? because i tried with 3 gmail accounts but it is showing login failed.

Yes, u can. but you have to make relevant changes in the code base, which is not preferred. I think you are having some connectivity issue. Because server is not showing your mail id: 
<img width="1371" alt="Screenshot 2019-05-14 at 11 50 15 AM" src="https://user-images.githubusercontent.com/43731599/57675133-a9c7bc00-763e-11e9-8498-3ad6b5836246.png">
